TestBenches/TrackMerger_test.cpp

```diff
@@ -20,40 +20,43 @@ int main(){
     
   // Input memories
   TrackFitType::TrackWord trackWord[kMaxTrack];
-  TrackFitType::BarrelStubWord barrelStubWords[kMaxTrack][4];
-  TrackFitType::DiskStubWord diskStubWords[kMaxTrack][4];
+  TrackFitType::BarrelStubWord barrelStubWords[kMaxTrack][6];
+  TrackFitType::DiskStubWord diskStubWords[kMaxTrack][5];
```

Comment:
@aryd Please use the constants from TrackletAlgorithm/Constants.h here:
```suggestion
  TrackFitType::BarrelStubWord barrelStubWords[kMaxTrack][trklet::N_LAYER];
  TrackFitType::DiskStubWord diskStubWords[kMaxTrack][trklet::N_DISK];
```

Reply:
Fixed.